config/rbac/role.yaml

```diff
@@ -99,3 +109,23 @@ rules:
   - triggerauthentications/status
   verbs:
   - '*'
+---
```

Comment:
why do we have this new, separate, role?

Reply:
Do you mean in other file? kubebuilder has generated it here, let me check if I can split them

Reply:
I have checked and the only way I have found is splitting the rbac generation for each service, setting which files should be used for each service instead of using `./...` . do you know other way to do it?

Reply:
I agree on splitting because this permission is added to the `keda-operator` role which is another component and this another identity.

Reply:
The question is, do we want the extra work of having to manage 3 different makefile targets for generating 3 different rbac files based on 3 different group of files (most probably, splitting the api files too)? I can give a try, but `kubebuilder` AFAIK doesn't support something like setting an arbitrary role name for each permission you define, it takes all the permissions and creates a single ClusterRole and Role with all of them.
Other option is to split it manually, but I don't like it at all because the CRDs and RBAC manifests are automatically generated

Reply:
I have split this and now webhooks component installation adds the extra permissions